src/GafferCycles/IECoreCyclesPreview/SocketAlgo.cpp

```diff
@@ -105,8 +107,10 @@ void setFloat2Socket( ccl::Node *node, const ccl::SocketType &socket, const IECo
 		default :
 			IECore::msg(
 				IECore::Msg::Warning, "Cycles::SocketAlgo",
-				boost::format( "Unsupported type `%1%` for socket `%2%` on node `%3%" )
-					% value->typeName() % socket.name % node->name
+				fmt::format(
+					"Unsupported type `{}` for socket `{}` on node `{}",
```

Comment:
It looks like we have either one too few, or one too many backtick characters around the last substitution here and on lines 83, 144, 179, & 219. The situation was the same in the previous formatting too, so I'll leave it up to you to decide how pedantic you feel... :)

Reply:
Did this as a followup rather than muck about prior to release : https://github.com/GafferHQ/gaffer/pull/5156.